### DISCUSSION TITLE
How to run shell use root permission

### DISCUSSION THREAD

**Discussion by fukai7722** [tauri-apps/tauri#5610] [Q&A]
I want run a system command shell  on the macOS
`sudo spctl --master-disable`
how to get the permission in tauri?

**Comment by jlarmstrongiv** [comment]
Same question, but as Administrator for Windows too

**Reply by jlarmstrongiv** [reply]
Discussion for windows https://github.com/tauri-apps/tauri/discussions/4201

**Comment by diwangtseb** [comment]
> I want run a system command shell on the macOS
> `sudo spctl --master-disable`
> how to get the permission in tauri?
first: update tauri.conf.json
{
  "name": "sudo",
  "cmd": "sudo",
  "args": true
}  
second:
take "spctl --master-disable" as parameters It works for me @fukai7722

**Comment by jlarmstrongiv** [comment]
Related discord [discussion](https://discord.com/channels/616186924390023171/1096449326672248903/1096449326672248903)
